uvloop/loop.pyx

```diff
@@ -3116,6 +3116,22 @@ cdef class Loop:
         await waiter
         return udp, protocol
 
+    def monitor_fs(self, path: str, callback, flags: int) -> asyncio.Handle:
```

Comment:
Same for `flags` here - let's make it 3 Python booleans `watch_entry`, `stat` and `recursive`.

Reply:
so on this one, the libuv documentation states the first two are not implemented on any backend, so just to avoid a un-fun surprise I only put in the recursive flag (bool)